low-level/framework/messaging/thread_controller.py

```diff
@@ -48,38 +51,158 @@
     RealStorActuatorMsgHandler
 from message_handlers.real_stor_encl_msg_handler import RealStorEnclMsgHandler
 from message_handlers.service_msg_handler import ServiceMsgHandler
+from framework.utils.store_factory import store
+
+module_persistent_data = {}
+module_cache_dir = os.path.join(DATA_PATH, "modules")
+node_id = Conf.get(GLOBAL_CONF, NODE_ID_KEY, "SN01")
+
+
+def _get_recovery_config(module_name):
+    """
+    Read sspl config for corresponding module recovery configs.
+
+    Common sensor recovery config will be overriden by individual
+    module recovery config.
+    """
+    recovery_count = Conf.get(
+        SSPL_CONF, f"{SSPL_LL_SETTING}>sensor_recovery_count", 0)
+    recovery_interval = Conf.get(
+        SSPL_CONF, f"{SSPL_LL_SETTING}>sensor_recovery_interval", 0)
+    # Override common recovery config if individual module has it
```

Comment:
this can come first & if module if specific config not there, then use common configs

Reply:
Yes, its as same as you recommend. Preference goes to module config first, then common config.
if module doesn't have expected configs, common configs will be used. If common config doesn't exist, it will use default values.